### PR TITLE
Fix typo in v3 and v4 sovereign clouds yaml pipeline issues

### PR DESCRIPTION
**Pull request by divyagandhisethi**
Releasing new Host versions.
[V3 Release](PASTE_V3_LINK_HERE)
[V2 Release](PASTE_V2_LINK_HERE)

---

### PR information

- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/development-docs/cleaning-up-commits.md).
- [X] If applicable, the PR references the bug/issue that it fixes in the description.
- [X] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

File: host/3.0/publish-appservice-stage-sovereign-clouds.yml [hunk 1]
```diff
@@ -145,7 +145,7 @@ steps:
           docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.8-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.8-appservice-$(CloudName)-stage${{stage}}
           docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.9-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.9-appservice-$(CloudName)-stage${{stage}}
 
-          docker push $TARGET_REGISTRY/python:$(TargetVersion)-appservice-$(CloudName-stage${{stage}}
+          docker push $TARGET_REGISTRY/python:$(TargetVersion)-appservice-$(CloudName)-stage${{stage}}
```
Comment (pragnagopa) [inline host/3.0/publish-appservice-stage-sovereign-clouds.yml:148]: what is the best way to test these changes before merging?

Comment (divyagandhisethi) [reply host/3.0/publish-appservice-stage-sovereign-clouds.yml:148]: Unfortunately there is no best way to test these yaml files without running the pipeline atleast once at the time of a release.

Comment (pragnagopa) [reply host/3.0/publish-appservice-stage-sovereign-clouds.yml:148]: Can we trigger release pipeline pointing to your branch?

**Review by pragnagopa — APPROVED** [review]
Approving but we need to figure out how do we test this out either via Unit tests script that @CooperLink  added recently or triggering on-demand from the PR branch

**Pull request merged**